CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ All notable changes to this project will be documented in this file.
 * Add `phel fmt` alias for format (#673)
 * Add support for numeric on `empty?` (#683)
 * Fix `interleave` allowing nil keys and values (#682)
+* Add `phel fmt` alias for format
+* Add support for numeric on `empty?`
```

Comment:
Both of these lines seem like duplications of previous lines (10, 11).

Reply:
upsi! the git conflict... thanks!